lib/intelligent_foods/api_client.rb

```diff
@@ -4,26 +4,29 @@ module IntelligentFoods
   class ApiClient
     attr_reader :id, :secret, :access_token
 
-    def initialize(id: nil, secret: nil, client: nil)
+    def initialize(id: nil, secret: nil, client: nil, access_token: nil)
```

Comment:
this looks like an orthogonal change that should go in it's own PR? which should probably also include some refactoring to this class (and/or an additional class) to better support the change.